lib/prototype-admin/reset.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Reset data - NHS prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {{ backLink({
+      "text": "Back",
+      "href": backHref
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+      <h1 class="nhsuk-heading-l">Reset data</h1>
+
+      <p class="nhsuk-body">This will reset all of the data entered in this session.</p>
+
+      <form action="/prototype-admin/reset-session-data" method="post">
+        <input type="hidden" name="previousPath" value="{{ backHref }}">
```

Comment:
Regarding the single `returnPath` idea

Here's how you POST but with `req.query` still containing `returnPath`

```suggestion
      <form action="/prototype-admin/reset-session-data?returnPath={{ returnPath }}" method="post">
```

Reply:
I always forget that you can do a POST with a query string! Feels wrong somehow, even though it works just fine.

Reply:
I've kept hidden field for now - but could revert to query string if needed. (Not sure it matters either way)